src/server/index.js

```diff
@@ -3,7 +3,7 @@ const logger = require('../logger');
 
 const createHealthcheckHandler = require('./healthcheck');
 
-function createServer(builderPool, buildSQSQueue, buildBullQueue) {
+function createServer(builderPool, queues) {
```

Comment:
With the bull board and the cloudwatch alerts for SQS, I am leaning toward getting rid of this.